Makefile

```diff
@@ -231,3 +231,20 @@ export KUSTOMIZE=$(GOBIN)/kustomize
 else
 export KUSTOMIZE=$(shell which kustomize)
 endif
+
+oc:
+ifeq (, $(shell which oc))
+	@{ \
+	set -e ;\
+	OC_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$OC_GEN_TMP_DIR ;\
+	export latest_oc_client_version=$$(curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ 2>/dev/null | grep -o \"openshift-client-linux-4.*tar.gz\" | tr -d \") ;\
```

Comment:
Is this really the best way to determine the latest version? Shouldn't we be using https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/release.txt instead?

Even so, we don't always want to be using the latest OCP binaries. While this is fine for `main`, we probably want to restrict ourselves to the target version of our current release.